Support/PowerShell.sublime-settings

```diff
@@ -0,0 +1,6 @@
+{
+	"word_wrap": false,
+	"translate_tabs_to_spaces": true,
+	// Skip `-` in word separators
+	"word_separators": "./\\()\"':,.;<>~!@#$%^&*|+=[]{}`~?",
```

Comment:
Interesting that you chose to omit the `-` character here. I'm wondering what the effect is on completions. Would it also make sense to omit the `$` character?

Reply:
Hmm. Good question about `$`. I think it could make sense.

The `-` is because the `Verb-Noun` format for functions. It's much more natural to double-click-select and get a whole function name than just half of it. (There's definitely a weird case when someone subtracts without spaces, though: `$foo-1`.)

I don't know of places that `$` is used outside of identifiers, so dropping it, too, could be the right thing. Do you have thoughts?

Reply:
> Do you have thoughts?

I don't have a strong opinion on this matter.

Reply:
I think I'll leave `$` in `word-separators` for now. At worst, the user is only typing one more character. And it prevents grouping the whole string with `"foo$bar"` or `$foo-$bar`.